tests/unittests/QuantizationTest.cpp

```diff
@@ -474,7 +513,8 @@ TEST(Quantization, enableRowwiseQuantizedFullyConnectedSymmetric) {
   auto *offsetsNode = llvm::dyn_cast<Constant>(rwNode->getOffsets().getNode());
   ASSERT_TRUE(offsetsNode);
 
-  // Because we're using symmetric quantization, the offsets should all be zero.
+  // Because we're using symmetric quantization, the offsets should all be
```

Comment:
Looks like from here down it's a lot of accidental changes by clang-format, can you revert these?

Reply:
Sure, I can revert it. Do you mean the current format is accepted and we don't need to run "./clang-format fix" ? Thanks!

Reply:
We do need to run clang-format still usually, but it looks like clang-format was run when there was an extra curly bracket somewhere, which made many comments reformat. I wanted to just keep the comments clean since it wasn't necessary. Looks good now I think, thanks!